### ISSUE TITLE
Twemoji support

### ISSUE THREAD

**Issue by mkljczk** [Big-Fig/Fediverse.app#68]
Mastodon web UI uses Twemoji by default, Soapbox uses them, I think Pleroma FE has them as well, having option to use them for consistent experience would be nice.

**Comment by xal** [comment]
Yeah, we plan to support different emoji packs, and flutter even have a related package - https://pub.dev/packages/twemoji . However, all texts in Fedi rendered through https://pub.dev/packages/flutter_html which looks like don't support custom emoji packs. So we need to fork flutter_html and integrate custom emoji support.
